poolscript/script.go

```diff
@@ -277,7 +277,9 @@ func (r *RecoveryHelper) LocateAnyOutput(expiry uint32,
 // For version 0 (p2wsh) this returns the hash of the following script:
 // <trader_key> OP_CHECKSIGVERIFY
 // <auctioneer_key> OP_CHECKSIG OP_IFDUP OP_NOTIF
+//
```

Comment:
Can we indent the whole script here so it can all be formatted as a single code block?